packages/apps/composer-app/src/proto/schema.proto

```diff
@@ -13,5 +13,4 @@ message ComposerDocument {
 
```

Comment:
1. we should now start to think about a types lib (similar to kai-types) where we have dxos types.
2. these types should not be app specific -- neither in namespace (dxos.app) -- nor name; i.e., this isn't a ComposerDocument; it's a Document that composer -- or kai -- uses. And would replace Document in kat-types.